code/Base/Hud/Party/PartyInvite.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using Sports.PartySystem;
+
+namespace Sports.UI;
+
+[UseTemplate]
+public class PartyInvite : Panel
+{
+	public Label InviteText { get; internal set; }
+	private Panel ProgressBar { get; set; }
+	public Client Client { get; internal set; }
+
+	TimeSince _received;
```

Comment:
Access modifier here?

Reply:
Also, this is the only place we seem to be using the underscore format. I think we should just use camelCase instead of _camelCase, since C# has no official standard and the _ is ugly.